client/src/routes/index.ts

```diff
@@ -1,16 +1,16 @@
 import { match } from "react-router";
 import { Dispatch } from "redux";
 
-import { ErrorPage } from "./pages/Error";
-import { SandboxPage } from "./pages/Sandbox";
-import { GamePage } from "./pages/Game";
-import { RegistrationPage } from "./pages/Registration";
-import { LoginPage } from "./pages/Login";
-import { ProfilePage } from "./pages/Profile";
-import { MenuPage } from "./pages/Menu";
-import { LeaderBoardPage } from "./pages/LeaderBoardPage";
-import { ForumPage } from "./pages/ForumPage";
-import { CommentsPage } from "./pages/CommentsPage";
+import { ErrorPage } from "../pages/Error";
+import { SandboxPage } from "../pages/Sandbox";
+import { GamePage } from "../pages/Game";
+import { RegistrationPage } from "../pages/Registration";
+import { LoginPage } from "../pages/Login";
+import { ProfilePage } from "../pages/Profile";
+import { MenuPage } from "../pages/Menu";
+import { LeaderBoardPage } from "../pages/LeaderBoardPage";
```

Comment:
здесь была переименована директория ранее :)
```suggestion
import { LeaderBoardPage } from "../pages/LeaderBoard";
```

Reply:
исправил